src/core/Akka/Actor/Exceptions.cs

```diff
@@ -134,12 +134,10 @@ public class ActorInterruptedException : AkkaException
     /// </summary>
     public class ActorInitializationException : AkkaException
     {
-        private readonly IActorRef _actor;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="ActorInitializationException"/> class.
         /// </summary>
-        protected ActorInitializationException()
+        public ActorInitializationException()
```

Comment:
public constructor is mandatory for NetCore support (due to limitations of our serializer)